TeamCode/src/main/java/org/firstinspires/ftc/teamcode/auton/DistanceSensorTest.java

```diff
@@ -0,0 +1,24 @@
+package org.firstinspires.ftc.teamcode.auton;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+import com.qualcomm.robotcore.hardware.DistanceSensor;
+
+import org.firstinspires.ftc.robotcore.external.navigation.DistanceUnit;
+
+@Autonomous(name="Distance Sensor Test")
+public class DistanceSensorTest extends OpMode {
+    DistanceSensor distanceSensor;
+    @Override
+    public void init() {
+        telemetry.addData("Initializing Distance Sensor.", null);
```

Comment:
The `telemetry.addData` method will appear in the format 
     `Initializing Distance Sensor.: null`

I think what you're looking for is the `telemetry.addLine` method, but if you want to use the `telemetry.addData` method you could do smt like 
     
     telemetry.addData("Distance Sensor", "Initializing");
     telemetry.addData("Distance Sensor", "Initialized"); 
     
which I think we've done a couple times as well

Reply:
if you're using the `telemetry.addData` though, get rid of the period